src/components/ChartCard/ChartCard.tsx

```diff
@@ -16,6 +17,7 @@ type IChartCardProps = { title: string, messageType?: 'user' | 'group'} & (
 
 
 const ChartCard: FC<IChartCardProps> = ({ categories, members, title, messageType = 'user' }) => {
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency);